kod/object/active/holder/room.kod

```diff
@@ -1790,6 +1790,20 @@ messages:
       return;
    }
 
+   RestartMusicAfterJala()
```

Comment:
Functions should be named for what they do, not who calls them.  Notice that this function doesn't restart anything, and has nothing to do with Jala.  One clue that this is a problematic name is that a generic Room shouldn't know anything about particular spell schools.

The danger of a name like this is that someone later notices that it actually just sends the current music to all users in the room.  Then they put a call to "RestartMusicAfterJala" in some code that has nothing to do with Jala, and it becomes super confusing.

Reply:
Fixed.